students/10th/solahn/westagram/account/views.py

```diff
@@ -0,0 +1,44 @@
+from django.shortcuts import render
+
+# Create your views here.
+import json
+from django.views import View
+from django.http import JsonResponse
+from .models import Account
+
+class SignUpView(View):
+   
+    
+    def post(self, request):
+        data = json.loads(request.body)
+        if Account.objects.filter(email = data['email']).exists():
+            return JsonResponse({'message':'email 중복'}, status=401)
+        else:
+            Account(
+                    email  = data['email'],
+                    password = data['password']
+                    ).save()
+            return JsonResponse({'message': '회원가입 완료'},status=200)
+
+    def get(self, request):
+        user_data = Account.objects.values()
+        return JsonResponse({'users':list(user_data)},status=200)
+
+    
+class SignInView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            if Account.objects.filter(email = data['email']).exists():
+                user = Account.objects.get(email = data['email'])
+                if user.password == data['password']:
+                    return JsonResponse({'message':f'{user.email}회원님 로그인 성공'}, status=200)
+                else:
+                    return JsonResponse({'message':'비밀번호 오류'}, status=401)
```

Comment:
비밀번호가 틀려서 인증이 안되는 오류라서 맞는 메시지(비밀번호 오류) 이지만 비밀번호 오류라고 에러메시지를 친절하게 보내주면 억지스럽긴 하지만 해커가 비밀번호만 오류인거를 알아서 반복적으로 시도한다거나 할 수 있기 때문에 UNAUTHORIZED 정도로 처리하면 될것 같습니다.
```suggestion
                    return JsonResponse({'message':'UNAUTHORIZED'}, status=401)
```